mne/event.py

```diff
@@ -746,7 +746,7 @@ def make_fixed_length_events(raw, id, start=0, stop=None, duration=1.,
         A raw object to use the data from.
     id : int
         The id to use.
-    start : float
+    start : int | float
```

Comment:
this change suggest a different behavior if start is int or float. Just keep float and cast to float the passed param below